pyproject.toml

```diff
@@ -1,9 +1,6 @@
 [tool.mypy]
 ignore_missing_imports = true
 no_implicit_optional = true
-plugins = [
-    "pydantic.mypy",
-]
```

Comment:
:eyes:

Reply:
Not needed anymore so prefer to remove it (we can always re-add it if we introduce pydantic again in the future)